commands/command_checkout.go

```diff
@@ -0,0 +1,111 @@
+package commands
+
+import (
+	"github.com/github/git-lfs/git"
+	"github.com/github/git-lfs/lfs"
+	"github.com/github/git-lfs/vendor/_nuts/github.com/spf13/cobra"
+	"os"
+	"os/exec"
+	"sync"
+)
```

Comment:
`script/fmt` prefers `goimports` (but falls back to `go fmt`) (#303). `goimports` (kind of annoyingly) has a slightly different way of ordering import blocks. It outputs this block as such:

```go
import (
	"os"
	"os/exec"
	"sync"

	"github.com/github/git-lfs/git"
	"github.com/github/git-lfs/lfs"
	"github.com/github/git-lfs/vendor/_nuts/github.com/rubyist/tracerx"
	"github.com/github/git-lfs/vendor/_nuts/github.com/spf13/cobra"
)
```

So when any of the scripts are run that call `script/fmt` (pretty much all of them) on a machine that has `goimports` installed, it'll end up changing the file.